database/seeds/DatabaseSeeder.php

```diff
@@ -30,22 +29,18 @@ public function run()
             // Create 10-20 signups with one accepted photo post & some pending photo and text posts.
             factory(Signup::class, rand(10, 20))->create(['campaign_id' => $campaign->id])
                 ->each(function (Signup $signup) use ($photoAction, $textAction) {
-                    $group = factory(Group::class)->states('school')->create();
-
                     $signup->posts()->save(factory(Post::class)->states('photo', 'accepted')->create([
                         'action_id' => $photoAction->id,
                         'signup_id' => $signup->id,
                         'campaign_id' => $signup->campaign_id,
                         'northstar_id' => $signup->northstar_id,
-                        'group_id' => $group->id,
                     ]));
 
                     $signup->posts()->saveMany(factory(Post::class, rand(2, 4))->states('photo', 'pending')->create([
                         'action_id' => $photoAction->id,
                         'signup_id' => $signup->id,
                         'campaign_id' => $signup->campaign_id,
                         'northstar_id' => $signup->northstar_id,
-                        'group_id' => $group->id,
```

Comment:
I thought you had just added these - why don't we want `group_id` anymore?

Reply:
I did! It'd be ideal to add these for a campaign with a group type. I think it'd be cleaner to add a separate `GroupSeeder` similar to this to create photo posts for schools/groups, which also generates action stats.

Reply:
Ah got it - that makes sense!